.gitignore

```diff
@@ -22,8 +22,8 @@ composer.lock
 tests/log
 
 # ignore core development artifacts
-./db
-./migrations
+/db
+/migrations
```

Comment:
I prefer a trailing slash to clarify folders vs files :)
But this is also fine for now

Reply:
Fixed, and added them to a handful of others, though I think I missed a handful of other folders.